### DISCUSSION TITLE
Ubuntu 14.04 LTS - Unable to find php5.6-fpm

### DISCUSSION THREAD

**Discussion by realtebo** [oerdnj/deb.sury.org#1991] [General]
I have an old 14.04 server (LTS), on a VPS,  which is using both

    ondrej-apache2-trusty.list
and
    ondrej-php-trusty.list

Still, I cannot find any candidate when trying to installa php5.6-fpm

What should I do ?

> No, I cannot upgrade my machine until programmers will not convert php5 code to php7. Them we willl move to a pretty new VPS

**Comment by sleemanj** [comment]
Invent a time machine and go back a decade?

You will need to compile yourself if you really must stay on 14.04, not even Freexian has releases for 14.04.

Otherwise you need to go to 20.04 for @oerdnj repository, which incidentally, does still support php 5.6, just not on antique ubuntu.

**Comment by oerdnj** [comment]
I can send you a quote to add 14.04 to Freedian, but most likely you will not like it.
